### PR TITLE
Refine and extend Node monitoring section of README

### PR DESCRIPTION
**Pull request by 31z4**
I removed https://orchestrator.strn.pl/stats because it redirects to https://dashboard.saturn.tech/stats, which is already mentioned. I also added links to the Saturn Explorer and Saturn Moonlet. The latter is the project that I built and maintain ☺️

### CODE REVIEW

**Comment by vorburger** [comment]
LGTM

**Comment by 31z4** [comment]
@DiegoRBaquero please have a look when you have time. Thanks!

**Comment by vorburger** [comment]
@31z4 FYI https://github.com/filecoin-saturn/L1-node/pull/481#discussion_r1307927403 _"Given Moonlet is external, it should not be mentioned in the official L1 FAQ"_ ... 😿

**Comment by 31z4** [comment]
@vorburger I think Diego's comment is relevant specifically to the FAQ section of the docs. In general, I think node operators should know about non-official monitoring tools built by the community.

File: README.md [hunk 1]
```diff
@@ -281,8 +281,10 @@ When payments are scheduled to be sent out, your Filecoin wallet will receive a
 
 ### Node monitoring
 
-- https://dashboard.saturn.tech - View historical data on your bandwidth contributions, FIL earnings, and more.
-- https://orchestrator.strn.pl/stats - View detailed, real-time stats on every Saturn node.
+- https://dashboard.saturn.tech - View detailed, real-time stats on every Saturn node.
+- https://dashboard.saturn.tech/address - View historical data on your bandwidth contributions, FIL earnings, and more.
+- https://explorer.saturn.tech - View a 3D geospatial visualization of the Saturn network, along with nodes and network statistics.
+- https://github.com/31z4/saturn-moonlet - Self-hosted Saturn monitoring using Prometheus and Grafana. View detailed, real-time and historical data on your nodes and earnings, setup alerts, and more.
```
Comment (hannahhoward) [inline README.md:287]:
```suggestion

### Community Tools

These Saturn tools are maintained by community members outside the Saturn core team.

- https://github.com/31z4/saturn-moonlet - Self-hosted Saturn monitoring using Prometheus and Grafana. View detailed, real-time and historical data on your nodes and earnings, setup alerts, and more.
```

@DiegoRBaquero does this resolve your concern. I'm fine with the change if it incorporates the disclaimer, but I don't want to override you.

@31z4,
Hey I'm @hannahhoward the (new-ish) Saturn tech lead. I am personally fine to merge with the above change to make it clear this is not a core team developed tool, if it resolves @DiegoRBaquero 's concern. My only other concern is -- are you actively maintaining and developing this tool? Do you think we can safely say that in six months we'll still be running it?

Comment (31z4) [reply README.md:287]: Hi @hannahhoward 👋 Yes, I'm going to continue maintain Saturn Moonlet and willing to build community around it.

**Comment by 31z4** [comment]
@hannahhoward I accepted your suggestion and updated Table of Contents. Also, @DiegoRBaquero approved already. Can you please merge now?

**Pull request merged**